client/pdo/client/commands/pservice.py

```diff
@@ -15,18 +15,26 @@
 import argparse
 import logging
 
+import pdo.common.config as pconfig
```

Comment:
Curious, at first glance the eservice.py, pservice.py and sservice.py look largely identical except some name ("patterened") differences and minor sub-command difference: Isn't there an easy way to unify them and remove code redundancy?

Reply:
see comment from above. again... yes and no. there is a lot of redundancy in the commands (less so in the support functions) but each type has different arguments. which is easier to understand? ridiculously complex arguments for one command or consistent arguments across multiple commands. in this case, i chose multiple commands. its a reasonable choice. there are other reasonable choices.